data/items/items.xml

```diff
@@ -2225,6 +2225,7 @@
 		<attribute key="showcharges" value="1" />
 		<attribute key="absorbPercentEnergy" value="10" />
 		<attribute key="showattributes" value="1" />
+		<attribute key="supply" value="true" />
```

Comment:
Note: It is correct to use 1 for true and 0 for false.
If false, it is not necessary to declare the item attribute.

Reply:
"true" is also correct for a boolean